pkg/api/v1/types.go

```diff
@@ -2292,6 +2292,12 @@ type Binding struct {
 	Target ObjectReference `json:"target"`
 }
 
+// Preconditions must be fulfilled before an operation (update, delete, etc.) is carried out.
+type Preconditions struct {
+	// Specifies the target UID.
+	UID *types.UID `json:"uid,omitempty"`
```

Comment:
I would need to see a use case for that, it sounds complicated.

On Fri, Mar 25, 2016 at 11:07 PM, Wojciech Tyczynski <
notifications@github.com> wrote:

> In pkg/api/v1/types.go
> <https://github.com/kubernetes/kubernetes/pull/22965#discussion_r57509961>
> :
>
> > @@ -2292,6 +2292,12 @@ type Binding struct {
> >  	Target ObjectReference `json:"target"`
> >  }
> >
> > +// Preconditions must be fulfilled before an operation (update, delete, etc.) is carried out.
> > +type Preconditions struct {
> > +	// Specifies the target UID.
> > +	UID *types.UID `json:"uid,omitempty"`
>
> Yeah - but I meant preconditions of RV of other objects (not the one that
> we are updating). This is something that you cannot currently do. But sure
> - we can add it later.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/22965/files/610daf2443db5029c745a27f1e693184ff9399dc#r57509961>
>